source/client-side-encryption/client-side-encryption.rst

```diff
@@ -699,40 +699,28 @@ Unencrypted data MUST NOT appear in the data of any command monitoring
 events. Encryption MUST occur before generating a CommandStartedEvent,
 and decryption MUST occur after generating a CommandSucceededEvent.
 
-Size limits and Wire Protocol Considerations
-============================================
-Because encryption increases the size of commands, the driver MUST
-instead use the following reduced size limits when constructing commands
-for encryption:
+Size limits for Write Commands
+==============================
+Automatic encryption requires the driver to serialize write commands as
+a single BSON document before automatically encrypting with libmongocrypt
+(analogous to constructing `OP_MSG payload type 0 <https://github.com/mongodb/specifications/blob/70628e30c96361346f7b6872571c0ec4d54846cb/source/message/OP_MSG.rst#sections>`_, not a document sequence).
+Automatic encryption returns a single (possibly modified) BSON document as the
+command to send.
 
--  maxBsonObjectSize = 2097152 bytes
--  maxMessageSizeBytes = 6000000 bytes
+Because automatic encryption increases the size of commands, the driver
+MUST split bulk writes at a reduced size limit before undergoing automatic
+encryption. The write payload MUST be split at 2MiB (2097152). Where batch
+splitting occurs relative to automatic encryption is implementation-dependant.
```

Comment:
"dependent"?

Reply:
Ah, not the only occurrence. One day I will learn to spell this word.